### PR TITLE
ER-1187 Aria landmark update

### PR DESCRIPTION
**Pull request by martikat**
[ER-1187](https://dfedigital.atlassian.net.mcas.ms/browse/ER-1187)

### CODE REVIEW

File: app/views/layouts/hero.html.slim [hunk 1]
```diff
@@ -33,19 +33,20 @@ html.govuk-template lang='en'
...
+          = yield :hero
 
-    .dfe-width-container
-      main#main-content.govuk-main-wrapper role='main'
+      .dfe-width-container class='govuk-!-padding-top-7 govuk-!-padding-bottom-7'
         = yield
-
-    = yield :other_resources
+    aside role='complementary'
+      = yield :shared_ctas
```
[the remark this answers is not in the document]
Comment (martikat) [reply app/views/layouts/hero.html.slim:49]: I have compared this to what we had before these changes and visually it looks the same. There has been some reorganising here to ensure the `aside` is in the correct place

File: app/views/pages/index.html.slim [hunk 2]
```diff
@@ -16,6 +16,7 @@
 - if page.slug == 'cookie-policy'
   = render 'settings/cookie_policy'
 
-= render 'shared/ctas'
-
-= render 'shared/other_resources'
+- content_for(:shared_ctas) do
```
Comment (peterdavidhamilton) [inline app/views/pages/index.html.slim:19]: The naming is confusing. It is the content for the `aside` which  contains...

Comment (peterdavidhamilton) [reply app/views/pages/index.html.slim:19]: And we'd agreed the content blocks would be defined first

**Review by peterdavidhamilton — CHANGES_REQUESTED** [review]

**Comment by peterdavidhamilton** [comment]
I think my previous comment still stands. How are you able to yield other resources?

**Review by peterdavidhamilton — APPROVED** [review]
Yeah, this looks more logical @martikat 👍🏼

**Pull request merged**